src/logging.cc

```diff
@@ -1153,9 +1153,8 @@ static LogMessage::LogMessageData fatal_msg_data_shared;
 // LogMessageData object exists (in this case glog makes zero heap memory
 // allocations).
 static GLOG_THREAD_LOCAL_STORAGE bool thread_data_available = true;
-static GLOG_THREAD_LOCAL_STORAGE std::aligned_storage<
-    sizeof(LogMessage::LogMessageData),
-    alignof(LogMessage::LogMessageData)>::type thread_msg_data;
+static GLOG_THREAD_LOCAL_STORAGE
+    char thread_msg_data[sizeof(void*) + sizeof(LogMessage::LogMessageData)];
```

Comment:
I would check whether the `alignof` keyword is available (using CMake) and if possible use it. Otherwise fallback to the C++98 solution.

Reply:
alignof keyword exists in c++11. Do you mind if I check c++ instead using standard convention, i.e.  
`__cplusplus >= 201103L`?

Reply:
Such checks are [unreliable](https://blogs.msdn.microsoft.com/vcblog/2018/04/09/msvc-now-correctly-reports-__cplusplus/). Please use CMake.

Reply:
Fixed